jobs/build/advisories/Jenkinsfile

```diff
@@ -0,0 +1,212 @@
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${buildUserEmail}"
+                        ],
+                        [
+                            name: 'IMPETUS',
+                            description: 'For which reason are those advisories being created',
+                            $class: 'hudson.model.ChoiceParameterDefinition',
+                            choices: [
+                                "standard",
+                                "cve",
+                                "ga",
+                                "test",
+                            ].join("\n"),
+                        ],
+                        [
+                            name: 'DATE',
+                            description: 'Format: YYYY-Mon-dd (i.e.: 2050-Jan-01)',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "${dateFormat.format(date)}"
+                        ],
+                        [
+                            name: 'DRY_RUN',
+                            description: 'Take no action, just echo what the job would have done.',
+                            $class: 'hudson.model.BooleanParameterDefinition',
+                            defaultValue: true
+                        ],
+                        commonlib.mockParam(),
+                        commonlib.suppressEmailParam(),
+                        [
+                            name: 'MAIL_LIST_SUCCESS',
+                            description: '(Optional) Success Mailing List\naos-cicd@redhat.com,aos-qe@redhat.com',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: "",
+                        ],
+                        [
+                            name: 'MAIL_LIST_FAILURE',
+                            description: 'Failure Mailing List',
+                            $class: 'hudson.model.StringParameterDefinition',
+                            defaultValue: [
+                                'aos-team-art@redhat.com'
+                            ].join(',')
+                        ],
+                        [
+                            name: 'SPECIAL_NOTES',
+                            description: '(Optional) special notes to include in the email',
+                            $class: 'hudson.model.TextParameterDefinition',
+                            defaultValue: ""
+                        ],
+                    ]
+                ],
+            ]
+        )
+
+        commonlib.checkMock()
+
+        currentBuild.description = ""
+        try {
+            stage("kinit") {
+                buildlib = load("pipeline-scripts/buildlib.groovy")
+                buildlib.kinit()
+            }
+            stage("create rpm advisory") {
+                cmd = "-g openshift-${params.VERSION} "
```

Comment:
This may just be personal preference, but in the interest of DRY and reducing punctuation, I prefer embedded blocks like:
```suggestion
                cmd = """
                    -g openshift-${params.VERSION} 
                    create -k rpm 
                    --assigned-to ${params.ASSIGNED_TO}
                    --manager ${params.MANAGER}
                    --package-owner ${params.PACKAGE_OWNER}
                    --impetus ${params.IMPETUS}
                    ${params.DATE ? "--date ${params.DATE} " : ""}
                    ${params.DRY_RUN ? "" : "--yes"}
                """
```
In case you're wondering, the library call strips out newlines.

Reply:
done :heavy_check_mark: